dask/bag/core.py

```diff
@@ -172,7 +172,12 @@ def map_partitions(self, func):
         dsk = dict(((name, i), (func, (self.name, i)))
                         for i in range(self.npartitions))
         return Bag(merge(self.dask, dsk), name, self.npartitions)
+    def reblock(self, blockdims):
+   
+        old_to_new = intersect_blockdims(self.blockdims, blockdims)
+        curry(reblock, old_to_new, blockdims)
 
+        self.map_partitions()
```

Comment:
You probably shouldn't be acting in the `dask/bag/` directory.  Probably you meant for this to be in `dask/array/core.py`